conduit/data/datasets/vision/nih.py

```diff
@@ -0,0 +1,186 @@
+from enum import Enum
+from pathlib import Path
+from typing import ClassVar, Optional, Union, cast
+
+import pandas as pd
+from ranzen import parsable, str_to_enum
+from sklearn.preprocessing import MultiLabelBinarizer
+import torch
+from typing_extensions import TypeAlias
+
+from conduit.data.datasets.utils import ImageTform
+from conduit.data.datasets.vision.base import CdtVisionDataset
+
+__all__ = ["NIHChestXRayDataset"]
+
+
+class NIHSplit(Enum):
+    train = "train_val_list.txt"
+    test = "test_list.txt"
+
+
+class NIHSensAttr(Enum):
+    GENDER = "Patient Gender"
+    AGE = "Patient Age"
+
+
+class NIHTargetAttr(Enum):
+    """
+    Taget attributes for the NIH chest x-ray dataset.
+
+    Fraction of labels that are positive for each of the thoracic diseases:
+        Atelectasis           0.103095
+        Cardiomegaly          0.024759
+        Consolidation         0.041625
+        Edema                 0.020540
+        Effusion              0.118775
+        Emphysema             0.022440
+        Fibrosis              0.015037
+        Hernia                0.002025
+        Infiltration          0.177435
+        Mass                  0.051570
+        Nodule                0.056466
+        Pleural_Thickening    0.030191
+        Pneumonia             0.012763
+        Pneumothorax          0.04728
+    """
+
+    ATELECTASIS = "Atelectasis"
+    CARDIOMEGALY = "Cardiomegaly"
+    CONSOLIDATION = "Consolidation"
+    EDEMA = "Edema"
+    EFFUSION = "Effusion"
```

Comment:
are we doing ALL_CAPS enums again?

Reply:
I think the plan was always to gradually transition to using them throughout conduit as all caps is the convention for enums

Reply:
However, do you think it'd be better to just remain internally consistent?

Reply:
No, as long as you have a plan, it's fine.